### PR TITLE
fix CVE-2021-23362, and document the process.

### PR DESCRIPTION
**Pull request by bhamail**
Fixing transitive dependency vulnerability, and documenting how to do it.

cc @bhamail / @DarthHater / @allenhsieh / @ken-duck

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,6 +296,79 @@ We chose to allow output directly to the stdout, so that the user can decide wha
...
 required. Connection problems with OSS Index will result in an exception.
 
+## How to Fix Vulnerabilities
+
+Or, "Patient: Hey Doc, it hurts when I do this. Doctor: Then don't do that."
+
+So you've found a vulnerability. Now what? The best case is to upgrade the vulnerable component to a newer/non-vulnerable
+version. However, it is likely the vulnerable component is not a direct dependency, but instead is a transitive dependency
+(a dependency of a dependncy, of a dependncy, wash-rinse-repeat). In such a case, the first step is to figure out which
+direct dependency (and sub-dependencies) depend on the vulnerable component. The `npm ls <vulnerable dependency>`
```
Comment (brent-spiner) [inline README.md:306]: You might provide the `yarn` command too, which is I think `yarn why <package>`

Comment (DarthHater) [inline README.md:306]: You might provide the yarn command too, which is I think yarn why <package>

Comment (bhamail) [reply README.md:306]: Yes! Done: 4712c46

**Review by DarthHater — APPROVED** [review]
LGTM!

**Pull request merged**